dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/WorkflowInstanceV2Controller.java

```diff
@@ -0,0 +1,129 @@
+package org.apache.dolphinscheduler.api.controller;
+
+import static org.apache.dolphinscheduler.api.enums.Status.*;
```

Comment:
we should avoid using `*` in our import

Reply:
is this change by you or by spotless itself?

Reply:
done.

Reply:
why this wildcard import still exists? cc @EricGao888

Reply:
@SbloodyS Current regex `import\s+[^\*\s]+\*;(\r\n|\r|\n)` missed this case `import static xxx;\n`. I will submit a fix for this. : )

Reply:
> @SbloodyS Current regex `import\s+[^\*\s]+\*;(\r\n|\r|\n)` missed this case `import static xxx;\n`. I will submit a fix for this. : )

Thanks.